docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
 
 For more information on authorization and access for ACK service controllers, including details regarding recommended IAM policies, see [Configure Permissions][configure-permissions].
 
+#### Optional step
```

Comment:
let's make it part of the regular steps. People might miss optional step and then wonder why studio is not working

Reply:
removed